### PR TITLE
CP-44311 Update Cert Kits forms to collect the changes partner made in test

### PR DESCRIPTION
**Pull request by catherineliuchina**

### CODE REVIEW

File: docs/xenserver-xencert-guide.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Published Oct 2023
 
 ### Overview  
 
-The purpose of this document is to familiarize the reader with XenServer Shared Storage Certification Kit. The certification kit is designed to certify the interoperability of various types of storage hardware with XenServer.  
+The purpose of this document is to familiarize the reader with XenServer Shared Storage Certification Kit. The certification kit is designed to certify the interoperability of various types of storage hardware with XenServer. XenServer certification kit requires run with XenServer latest veriosn, ensure that XenServer is updated to the latest version before test.
```
Comment (Zhengchai) [inline docs/xenserver-xencert-guide.md:56]: Hi, 
"requires run with XenServer latest version" - this is not correct in general.
At the moment, each XS certification kit release runs against its mapped XenServer release, e.g. 8.2 CU1 cert kits run to the latest of 8.2 CU1, and XS 8 cert kits run with the latest XS 8 release.
We can release XS cert kits with backward comp-abilities (i.e. release cert kits as separated deliverable and make it run with all XS releases), however, that needs to be verified with all released XS releases, and this also needs to change the HCL website's release page (there has per release based Cert Kits at the moment)
 #

File: docs/xenserver-shared-storage-verification-form.docx
Comment (Zhengchai) [inline docs/xenserver-shared-storage-verification-form.docx:1]: I didn't see any tracked changes on this docx file, please use "word tracking" to indicate what has been changed for review, thanks

**Review by stephenchengCloud — APPROVED** [review]

File: docs/xenserver-xencert-guide.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Published Oct 2023
 
 ### Overview  
 
-The purpose of this document is to familiarize the reader with XenServer Shared Storage Certification Kit. The certification kit is designed to certify the interoperability of various types of storage hardware with XenServer.  
+The purpose of this document is to familiarize the reader with XenServer Shared Storage Certification Kit. The certification kit is designed to certify the interoperability of various types of storage hardware with XenServer. The XenServer8 certification kit needs to be run with the latest version of the corresponding XenServer Release. Make sure that XenServer8 has been updated to the latest version before testing.
```
Comment (xihuan-citrix) [inline docs/xenserver-xencert-guide.md:55]: XenServer8 -> XenServer 8?

File: docs/xenserver-xencert-guide.md (hunk 1 above)
Comment (xihuan-citrix) [reply docs/xenserver-xencert-guide.md:56]: Agree, this is specific for XenServer 8 release

Comment (xihuan-citrix) [reply docs/xenserver-xencert-guide.md:56]: "Make sure that XenServer8 has been updated to the latest version before testing. " this seems not necessary if we already said latest XenServer 8 release previously.

**Review by xihuan-citrix — APPROVED** [review]

**Comment by xihuan-citrix** [comment]
I see the "XenServer 8 Stream XenCert Verification Form", and I see in GCK you delete the "Stream", may need to keep the same.

**Pull request merged**